cli/functionary/login.py

```diff
@@ -17,6 +17,6 @@ def login_cmd(ctx, user, password, host):
     for other functionary commands to use to communicate with the server.
     """
     save_config_value("host", host)
-    token = login(user, password)
+    token = login(f"{host}/api/v1/api-token-auth", user, password)
```

Comment:
What I meant here was just pass the host in.  So instead of taking a url, `login` will take a host parameter.  Then construct the url inside of the login function.